composeApp/src/commonMain/kotlin/org/ooni/probe/ui/descriptor/add/AddDescriptorScreen.kt

```diff
@@ -131,18 +136,37 @@ fun AddDescriptorScreen(
                         modifier = Modifier.weight(1f),
                     )
                 }
-                LazyColumn {
+            }
+
+            Box(
```

Comment:
Why is this Box needed?

Reply:
Adding the box to ensure the actions are displayed at the bottom of the screen and not at the end of the URL list.
This ensures we can install the descriptor without having to scroll to the end of the list when its a long one.